airflow/operators/dagrun_operator.py

```diff
@@ -20,12 +20,19 @@
 from typing import Dict, Optional, Union
 
 from airflow.api.common.experimental.trigger_dag import trigger_dag
-from airflow.models import BaseOperator, DagRun
+from airflow.models import BaseOperator, DagRun, BaseOperatorLink
 from airflow.utils import timezone
 from airflow.utils.decorators import apply_defaults
 from airflow.utils.types import DagRunType
 
 
+class TriggerDagRunLink(BaseOperatorLink):
+    name = 'Triggered DAG'
+
+    def get_link(self, operator: "TriggerDagRunOperator", dttm):
+        return f"/graph?dag_id={operator.trigger_dag_id}&root=&execution_date={dttm}"
```

Comment:
Can you escape dttm? This may contain characters not allowed in the URL.

Reply:
Good call,

We can do (or something like that):

```
quote(self.execution_date.isoformat())
```